Go/Go.sublime-syntax

```diff
@@ -118,26 +121,72 @@ contexts:
     #
     # //go:some_directive arg0 arg1 ...
     #
+    # These have been adopted outside the gc compiler for use in linting
+    # directives and pragmas for other compiler suites where they take the
+    # more generalised form of:
+    #
+    # //namespace:some_directive ...
+    #
     # They're not part of the language spec, may not be recognized by some Go
     # compilers, and may stop working in future releases. Therefore,
     # highlighting them as compiler pragmas could be misleading. We scope them
     # as plain comments by default, and add some detailed meta scopes for
     # enterprising users wishing to add more color.
-    - match: (//)(go)(:)({{ident}})?
+    - match: (//)([a-z]+[a-z-]*)(:)({{directive}})[ ]?
       captures:
         1: punctuation.definition.comment.go
         2: meta.keyword.annotation.go
         3: meta.punctuation.accessor.colon.go
         4: meta.variable.function.go
       push:
         - meta_scope: meta.annotation.go comment.line.go
+        - match: .+
+          scope: meta.variable.parameter.go
```

Comment:
How about ...

```suggestion
        - meta_content_scope: meta.variable.parameter.go        
```

Otherwise should it use the same pattern as in line 172?

```suggestion
        - match: \S+
          scope: meta.variable.parameter.go
```